lms/static/js/discovery/main.js

```diff
@@ -0,0 +1,22 @@
+RequireJS.require([
+    'jquery',
+    'backbone',
+    'js/discovery/app',
+    'js/discovery/collection',
+    'js/discovery/form',
+    'js/discovery/result_list_view',
+    'js/discovery/filter_bar_view',
+    'js/discovery/search_facets_view'
+], function ($, Backbone, App, Collection, DiscoveryForm, ResultListView, FilterBarView, FacetsBarView) {
```

Comment:
Do we get a quick win if we concatenate the multiple .js files? We can't deliberately contribute slow code, what degree of performance change do we expect on a high-latency connection (such as for a user in China)?

Changes to LMS cache-busting are out-of-scope for this work. If we need to adapt the LMS to use a different strategy, that requires a separate story.